cibuild.cmd

```diff
@@ -145,5 +147,15 @@ exit /b 1
 @REM Kill any instances of msbuild.exe to ensure that we never reuse nodes (e.g. if a non-roslyn CI run
 @REM left some floating around).
 
-taskkill /F /IM vbcscompiler.exe 2> nul
-taskkill /F /IM msbuild.exe 2> nul
+@REM An error-level of 1 means that the process was found, but could not be killed.
+taskkill /F /IM vbcscompiler.exe > nul
+if %ERRORLEVEL% == 1 GOTO ActuallyFail
+
+taskkill /F /IM msbuild.exe > nul
+if %ERRORLEVEL% == 1 GOTO ActuallyFail
+
+taskkill /F /IM csc.exe > nul
+if %ERRORLEVEL% == 1 GOTO ActuallyFail
```

Comment:
These lines have been removed.  Do you want me to add the "is csc/vbc running" check?